examples/acquiring_data/downloading_hmi.py

```diff
@@ -1,67 +0,0 @@
-"""
```

Comment:
Unsure how I feel about deleting this in the backport.

I am tempted to restore it.